db/migrations/000001_create_activities_table.up.sql

```diff
@@ -0,0 +1,25 @@
+BEGIN;
+
+CREATE TABLE activities
+(
+    id              bigserial       CONSTRAINT activities_pk PRIMARY KEY,
+    uuid            uuid            DEFAULT uuid_generate_v4() NOT NULL,
+    fork_id         uuid            NULL,
```

Comment:
(non-blocker): Not sure if `fork_uuid` should be here or if it makes more sense to keep it in the `activity_lesson`

Reply:
You can fork a Syllabus, right? And change the Syllabus, adding or removing Lessons. We should keep `fork_id` here to track the Syllabus that was forked.

Reply:
yeap, i know, kind of agree, but i also think it depends on how we are going to use this information.  The fork would be in the syllabus entity, the content and the way the activity and lessons were organized. The user can make a series of changes to the syllabus after do the fork, I wonder if it will be too complicated to tracking or lose historical information over the changes. I'm starting to think that even in activy_lesson would be wrong. Maybe we need to better define the scope of this feature, i dont know.

Reply:
I don't have any strong opinions here if we need to discuss more, it's fine. Also, now that I was looking at my comment, I was talking about the syllabus instead of the activity, and well, "uma coisa é uma coisa, outra coisa é outra coisa"... I guess that I messed up, 'cause I was thinking about the syllabus, but even so, allowing users to fork activity is something good, but maybe the name "fork" is confusing us as SE, it is more like "clone" or "duplicated"...

Reply:
I just removed the reference. Let's schedule a meeting to discuss this and add changes to another PR